test/orchard/stacktrace/parser_test.clj

```diff
@@ -0,0 +1,43 @@
+(ns orchard.stacktrace.parser-test
+  (:require
+   [clojure.test :refer [deftest is testing]]
+   [orchard.stacktrace.parser :as parser]
+   [orchard.stacktrace.parser.test :as test]))
+
+(deftest parse-test
+  (doseq [fixture test/fixtures]
+    (testing (format "parse fixture %s" fixture)
+      (let [{:keys [cause error trace]} (parser/parse (test/read-fixture fixture))]
+        (testing "should succeed"
+          (is (nil? error)))
+        (testing "should parse the cause"
+          (is (string? cause)))
+        (testing "should parse the trace"
+          (is (every? test/stacktrace-element? trace)))))))
```

Comment:
Please prefer this:

```clj
(doseq [t trace]
  (is (test/stacktrace-element? trace)
      t))
```

`every?` fails opaquely in comparison - we should be reported of the specific item that failed

Reply:
Fixed